indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1String.kt

```diff
@@ -18,7 +18,7 @@ sealed class Asn1String : Asn1Encodable<Asn1Primitive> {
     @Serializable
     @SerialName("UTF8String")
     class UTF8(override val value: String) : Asn1String() {
-        override val tag = BERTags.UTF8_STRING
+        override val tag = BERTags.UTF8_STRING.toULong()
```

Comment:
imagine if this language had useful implicit integer conversion rules, like the kind that was invited, *checks*

![image](https://github.com/user-attachments/assets/e42b322b-fe47-4d25-8338-cec1eb20a881)

... the kind that was invented roughly _fifty-two years ago_.